### DISCUSSION TITLE
手機市占率

### DISCUSSION THREAD

**Discussion by Ryanlolry** [1101-GS4719-C/Python-Community-GS4719-C#1645] [General]
從這裡可以了解到蘋果在台灣市佔率約三成，而三星也是大約三成，但我認為兩者的差異是蘋果手機上榜的屬於高階機，但三星的有兩隻屬於中低階機。也可以從表格中發現台灣人偏好中低階機(5000-15000元左右)，所以可以統整出台灣人除了蘋果與三星的高階機系列外，會必較偏好購買較低規格的手機。
參考資料:
https://tw.news.yahoo.com/2023%E5%B9%B47%E6%9C%88%E6%89%8B%E6%A9%9F%E5%93%81%E7%89%8C%E5%B8%82%E4%BD%94%E6%8E%92%E8%A1%8C%E5%87%BA%E7%88%90-%E8%98%8B%E6%9E%9C%E7%AC%AC-30-3-%E4%B8%89%E6%98%9F%E7%B7%8A%E8%BF%BD%E5%9C%A8%E5%BE%8C-095242584.html

**Comment by Yanxxx7979** [comment]
感覺一部分中低階機很有可能是作為公務機或是備用機被大量使用。

**Comment by Terryyyyyyyyyyyy** [comment]
有人會把中階手機當主力機使用，畢竟在我看來並不是所有人都需要強大的cpu，照片拍攝與AI運算，所以如果選擇中低階機反而可以省下一大筆錢去購買不需要的功能。

**Comment by Chengbosco** [comment]
我以前也都是買中低階手機(紅米)，但轉蘋果之後是 真的好用，之前的紅米一開始還很正常但過一年就發燙常沒電，現在i11用了四年都算正常，不得不說是真的好用，而且蘋果的共通性也做得很好，在同儕之間非常方便。

**Comment by uwje** [comment]
高階機不管是在各種的耗材使用，還是軟體的配合，抑或是處理器的使用，都來得比中低階機來的好，所以我認為買高階級其實會買到很多根本用不到的功能，但衝著高階級的優良品質或品牌價值，所以會有很多人選擇高階機。

**Comment by uwje** [comment]
雖然我也是旗艦手機的用戶，我反而會去找相同規格但價格較平價的手機品牌，但我仍不會選擇三星及中國手機，前者我是有怕他的穩定度，後者是怕他的安全性。所以我選擇台灣的華碩，雖然說以外界評價來說不是太好，但以我自己的使用來說是正常的。以相同的處理器規格來說，他是我認為最有cp值的。
